README.md

```diff
@@ -30,6 +30,7 @@
   - [Set a Privacy Manager Id for the Property Group](#set-a-privacy-manager-id-for-the-property-group)
   - [ProGuard](#ProGuard)
   - [Programmatically consenting the current user](#Programmatically-consenting-the-current-user)
+  - [Programmatically delete the saved consent](#Programmatically-delete-the-saved-consent)
```

Comment:
What if we merge those 2 sections into one called:
### Adding/Removing custom consents